### DISCUSSION TITLE
@entangle did not work

### DISCUSSION THREAD

**Discussion by hussain-alkhatib** [livewire/livewire#6710] [7. Bugs (archived)]
### Livewire version

v 3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

in livewire 2 i was use 
users_query: $persist(@entangle('query'))
so when user make search it save the value of search in local storage using $persist
and when he close browser or refresh the page the value of search field taken from local storage and it will automatic search 
that mean it send the value from local storage to component using @entangle
but in livewire 3 
when reload the page it just take default value of query property from livewire component and set it to users_query in local storage.
i try to use users_query: $persist(@entangle('query').live) as docs and didnot work too.
how can fix that?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by artworkad** [comment]
There is already a discussion for this https://github.com/livewire/livewire/discussions/6271
